causalpy/tests/test_integration_skl_examples.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+import pytest
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.gaussian_process.kernels import ExpSineSquared, WhiteKernel
+from sklearn.linear_model import LinearRegression
+
+import causalpy as cp
+
+
+@pytest.mark.integration
+def test_did():
+    data = cp.load_data("did")
+    assert type(data) is pd.DataFrame
```

Comment:
I think is better to use (apply to all of the above where `type(...)` is used)
```python
assert isinstance(data, pd.DataFrame)
```

Reply:
Sure. That's done now